Simplenote/src/main/java/com/automattic/simplenote/NotesActivity.java

```diff
@@ -1411,7 +1415,7 @@ public void onConfigurationChanged(@NonNull Configuration newConfig) {
                 }
 
                 invalidateOptionsMenu();
-            // Go to NoteEditorActivity if note editing was fullscreen and orientation was switched to portrait
+                // Go to NoteEditorActivity if note editing was fullscreen and orientation was switched to portrait
```

Comment:
```suggestion
            // Go to NoteEditorActivity if note editing was fullscreen and orientation was switched to portrait
```

This comment was indented with the checkstyle automation. Let's move it back one indention so that it is the way it was before this pull request and it is clearer the comment is referring to the `else if` below.